app/services/custom_landing_page/landing_page_store_static.rb

```diff
@@ -10,8 +10,15 @@ def released_version(*)
     end
 
     def load_structure(*)
-      LandingPageStoreDefaults.add_defaults(
+      structure = LandingPageStoreDefaults.add_defaults(
         JSON.parse(CustomLandingPage::ExampleData::DATA_STR))
+      
+      begin
```

Comment:
don't use exceptions for flow control

Reply:
Changed flow control to use Module#const_defined? in 2451599639851d7721dc02dfc78078e7bdcabdf9